src/canvg.js

```diff
@@ -917,7 +917,7 @@ function build(opts) {
       if (this.style('mask').hasValue()) { // mask
         var mask = this.style('mask').getDefinition();
         if (mask != null) mask.apply(ctx, this);
-      } else if (this.style('filter').hasValue()) { // filter
+      } else if (this.style('filter').hasValue() && this.style('filter').value && this.style('filter').value !== 'none' && this.style('filter').value !== 'undefined') { // filter
```

Comment:
it should never be undefined as a string, so i'd rather you do something like `this.style('filter').valueOrDefault('none') != 'none'` ... or just check for `'none'` on the next line.

Reply:
Hi, thanks for the feedback, code has changed to use valueOrDefault.